src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -40,6 +50,21 @@ public function dump(MessageCatalogue $messages, $options = array())
     }
 
     /**
+     * Set XLIFF tool element metadata.
+     *
+     * @link http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#tool_elem
+     *
+     * @param array attributes supported by the <tool> element
+     */
+    protected function setToolMetadata( array $meta ){
```

Comment:
@timwhitlock we start with marking methods private and only open them if someone with a good use case asks for it.